clang/lib/Headers/hlsl/hlsl_intrinsics.h

```diff
@@ -1297,5 +1297,10 @@ _HLSL_AVAILABILITY(shadermodel, 6.0)
 _HLSL_BUILTIN_ALIAS(__builtin_hlsl_wave_active_count_bits)
 uint WaveActiveCountBits(bool Val);
 
+/// \brief Returns the index of the current lane within the current wave.
+_HLSL_AVAILABILITY(shadermodel, 6.0)
+_HLSL_BUILTIN_ALIAS(__builtin_hlsl_wave_get_lane_index)
+uint WaveGetLaneIndex();
```

Comment:
We should probably also apply `__attribute__((convergent))` to this function declaration. I know we didn't do that on the `WaveActiveCountBits` declaration, but we also didn't properly implement codegen for that.

Reply:
Yes, good catch, done